src/services/twilio-api/twilio_pricing.json

```diff
@@ -219,34 +201,13 @@
             "items": {
               "properties": {
                 "base_price": {
-                  "anyOf": [
-                    {
-                      "type": "number"
-                    },
-                    {
-                      "type": "null"
-                    }
-                  ]
+                  "type": "number"
```

Comment:
Do we need the "nullable" key for this one (and the other endpoints below)? 

I see that similar modified endpoints that had `"type": "null"` are now translated to `"nullable": true`.

Same may be true for the rest of the cases below.

Reply:
I don't _think_ so. None of the other price object schemas allow nulls.